### DISCUSSION TITLE
error 4723 and 9795

### DISCUSSION THREAD

**Discussion by Czepski** [farag2/Sophia-Script-for-Windows#137] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/76943895/109468166-33ac7780-7aa7-11eb-839d-85f363cc19a9.png)
ive tried turing off all firewalls and windows defender. strill doing the same thing. how can i make this work?

**Comment by Czepski** [comment]
the only problem i can see is that i have run O&O shutup10 a day before this.

**Reply by farag2** [reply]
According to the errors O&O shutup10 isnt the problem. may be... But I'm not confess with the service error. As you can see it has been removed.

**Comment by farag2** [comment]
What do you get?

```powershell
(New-Object -ComObject Microsoft.Update.ServiceManager).Services | Where-Object {$_.ServiceID -eq "7971f918-a847-4430-9279-4a52d1efe18d"}
```

**Reply by Czepski** [reply]
type that indo cmd or powershell? in powershell it continues to another line

**Reply by farag2** [reply]
PowerShell of course...

**Reply by Czepski** [reply]
right. nothing. it just goes to the next line..
you mean run this along with the other code and then run the script?

**Reply by farag2** [reply]
I mean let's move to Discord to discuss...

**Reply by Czepski** [reply]
still getting it 
![image](https://user-images.githubusercontent.com/76943895/109491071-1639d680-7ac4-11eb-9ac4-b304f9c71b4d.png)

**Reply by Czepski** [reply]
which discord?

**Reply by farag2** [reply]
> > because i have used a free key on an open download service allowed by microsoft
> 
> M, what?.. What kind of key? So could you write to my Discord? sanctuary_s#6351 I can connect to you via TeamViewer for example.

I wrote to you some messages before: my Discord. sanctuary_s#6351

**Comment by farag2** [comment]
> Cannot find path bla-bla .bmp\ShellNew

This registry key was already removed. So you cannot remove it twice )

**Comment by farag2** [comment]
I found it. Navigate in the app to the "Windows Update" section — Disable Windows Updates for other products. Enable it.
it can be cnfigured via Windows 10 Settings: `Settings — Update and Security — Windows Update — Advanced Options — Receive updates for other Microsoft products when you update Windows`.

**Reply by Czepski** [reply]
'your organisation' has turned off automatic uodates 
and then in advanced options:
![image](https://user-images.githubusercontent.com/76943895/109489363-b6423080-7ac1-11eb-810a-01face80f430.png)

**Reply by farag2** [reply]
It's made by O&O shutup10. Find the button I wrote and enable it.

**Reply by Czepski** [reply]
because i have used a free key on an open download service allowed by microsoft

**Reply by farag2** [reply]
> 
> 
> because i have used a free key on an open download service allowed by microsoft

M, what?.. What kind of key? So could you write to my Discord? sanctuary_s#6351 I can connect to you via TeamViewer for example.

**Comment by farag2** [comment]
I tested it: it has no effect. LOL. nice app )

**Reply by Czepski** [reply]
Thankyou mate. fixed it. and i learnt a  bit in the process 
![image](https://user-images.githubusercontent.com/76943895/109496189-48026b80-7acb-11eb-9f43-f8797bb11c08.png)
as earlier stated that error is  because it cant remove something thats already been removed

**Comment by Czepski** [comment]
Thankyou mate. fixed it. and i learnt a  bit in the process 
![image](https://user-images.githubusercontent.com/76943895/109496189-48026b80-7acb-11eb-9f43-f8797bb11c08.png)
as earlier stated that error is  because it cant remove something thats already been removed.

**Reply by Czepski** [reply]
i should  quickly add... if you have policies you have to delete them in your local machine registry software>policies>microsoft>windows>windows update otherwise you might not get the latest update and then the script might not work the way it was intended to run with the latest update

**Reply by farag2** [reply]
And read this ))
https://github.com/farag2/Windows-10-Sophia-Script/blob/047271b84215c965def8453e05865bd4367f275c/Sophia/PowerShell%205.1/Sophia.psm1#L15

Could add more: check you applied policies in
`HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies`
`HKLM:\SOFTWARE\Policies\Microsoft\Windows`
`HKCU:\SOFTWARE\Policies\Microsoft\Windows`
and never use such apps like ShutUp10 if you do not understand what they do. 🥇

**Comment by Czepski** [comment, marked as answer]
Thankyou mate. fixed it. and i learnt a bit in the process
![image](https://user-images.githubusercontent.com/76943895/109498281-379fc000-7ace-11eb-9246-2f7aab9e51f8.png)
as earlier stated that error is because it cant remove something thats already been removed.
how do i say this is answered?
